src/GafferVDB/MeshToLevelSet.cpp

```diff
@@ -184,66 +188,102 @@ const FloatPlug *MeshToLevelSet::interiorBandwidthPlug() const
 	return getChild<FloatPlug>( g_firstPlugIndex + 3 );
 }
 
-bool MeshToLevelSet::affectsProcessedObject( const Gaffer::Plug *input ) const
+bool MeshToLevelSet::affectsMergedObject( const Gaffer::Plug *input ) const
 {
 	return
-		ObjectProcessor::affectsProcessedObject( input ) ||
+		MergeObjects::affectsMergedObject( input ) ||
 		input == gridPlug() ||
 		input == voxelSizePlug() ||
 		input == exteriorBandwidthPlug() ||
 		input == interiorBandwidthPlug()
 	;
 }
 
-void MeshToLevelSet::hashProcessedObject( const ScenePath &path, const Gaffer::Context *context, IECore::MurmurHash &h ) const
+void MeshToLevelSet::hashMergedObject(
+	const ScenePath &path, const Gaffer::Context *context, IECore::MurmurHash &h
+) const
 {
-	ObjectProcessor::hashProcessedObject( path, context, h );
+	MergeObjects::hashMergedObject( path, context, h );
```

Comment:
It's unfortunate that `MergeObjects::hashMergedObject()` includes the hash of the input transform. This means that in the simple case of converting a single mesh with the default destination, we now repeat the compute every time the transform changes. This is quite a noticeable performance difference, so I wonder if we could detect this case in the base class, and omit the transform from the hash?